uisdk/src/main/res/values/strings.xml

```diff
@@ -397,4 +397,11 @@
     <!--  Loyalty  -->
     <string name="kh_uisdk_loyalty_balance_title">BALANCE: %s POINTS</string>
     <string name="karhoo_uisdk_locale">en_GB</string>
+
+    <!-- Legal Notice - The strings can be translated by DP -->
+    <string name="kh_uisdk_legal_notice_label">Legal Notices</string>
+    <string name="kh_uisdk_legal_notice_title" translatable="false" />
```

Comment:
The key should be `translatable`, can we add it to all locales? the value would be empty though. 
Did you add the `translatable="false"` or it was added by phrasee.com?

Reply:
The translatable="false" is only for the linter to not show a warning
Phrase removes empty strings from the locale files, so in order to not need to add them manually at each strings updates, i've added the translatable flag

Reply:
Found a way to export also the empty keys from Phrase, so i've removed the translatable flag